PRINCIPLES.md

```diff
@@ -15,22 +16,32 @@ Approved by GB on: Dec 5, 2017
 
 ## TOC Operating Principles
 
-Now that CNCF has been active for over a year we want to start writing down what we have learned. Future TOCs can still make changes, but there will at least be documented precedent.
+ As part of our continued community engagement and iterations on the work done, the following list of principles serve to provide an understanding of how the TOC intends to work with projects, effectively the guiding star the the TOC in their work. It also provides insight into what the TOC is looking for continuity and evolution across each seated TOC. Since the CNCF's inception we have endevoured to capture the lessons learned and interate on these principles to service the foundation, the community, and the projects.
+
+ 1. We are project-centric.
+ 1. We exercise minimal viable governance to enable community driven success.
+ 1. Do no harm by enabling robust, well engineered projects and supportive processed to be leveraged for greatest adoption likelihood.
```

Comment:
- `supportive processed` -> `support processes`
- `be leveraged for greatest adoption likelihood` -> `to increase the likelihood of success`